### ISSUE TITLE
Usage on android

### ISSUE THREAD

**Issue by Y-E-P** [waldheinz/fat32-lib#28]
How I can use this library on Android? Has any examples?

**Comment by waldheinz** [comment]
I have little experience with Android development, but I don't know a reason why it wouldn't work. Is there any particular problem?

**Comment by Y-E-P** [comment]
yep,if I understand correctly, if I need to read a fat12 disk i need to create FileDisk object and passed to constructor the path to disk,right?
